libs/ledger-live-common/src/e2e/speculos.ts

```diff
@@ -358,7 +358,7 @@ export async function pressBoth() {
 
 export async function pressUntilTextFound(
   targetText: string,
-  maxAttempts: number = 10,
+  maxAttempts: number = 15,
```

Comment:
why was this number increased?

Reply:
I had to increase the number bc for Algorand the "Accept" appears on the 11 screen of the speculos